pkg/sentry/projects.go

```diff
@@ -35,6 +36,25 @@ func (sc *SentryClient) GetProjects(organizationSlug string) ([]SentryProject, e
 	return out, err
 }
 
+// Fetch all projects, checking if a link for the next page is specified in the headers
+func (sc *SentryClient) GetAllProjects(organizationSlug string) ([]SentryProject, error) {
+	allProjects := []SentryProject{}
+	url := "/api/0/organizations/" + organizationSlug + "/projects/"
+
+	for (url != "") {
+		projects := []SentryProject{}
+		nextURL, err := sc.FetchWithPagination(url, &projects)
```

Comment:
It doesn't look like this API handles rate limits - should we wrap this in an exponential backoff? Perhaps we should do this in a separate PR for all endpoints another time? 😄

Reply:
:+1: Maybe you can wrap request sending logic in a separate package and use a decorator pattern to add rate limits check, backoff and retry logics?
I agree it is better to be a separate PR.

Reply:
I'm planning to create a separate PR for the rate limiting handling. We have `sentryHttpClient`, so I'm planing to put the login there.